commands/hugo.go

```diff
@@ -77,10 +77,6 @@ func newUserError(a ...interface{}) commandError {
 	return commandError{s: fmt.Sprintln(a...), userError: true}
 }
 
-func newUserErrorF(format string, a ...interface{}) commandError {
```

Comment:
It is not used, but it completes a natural four method API, and it is just a question of time until it is getting used, so don't remove it.